src/plugins/home/public/application/components/homepage/sections/recent_work.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import moment from 'moment';
+import { EuiFlexItem, EuiCard, EuiIcon, EuiFlexGrid } from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import { ChromeRecentlyAccessedHistoryItem } from 'opensearch-dashboards/public';
+import { Section } from '../../../../services/section_type/section_type';
+import { renderFn } from './utils';
+import { getServices } from '../../../opensearch_dashboards_services';
+
+import '../_homepage.scss';
+
+const itemType = [
+  {
+    type: 'dashboard',
+    name: 'Dashboard',
+    icon: 'dashboardApp',
+  },
+  {
+    type: 'visualization',
+    name: 'Visualization',
+    icon: 'visualizeApp',
+  },
+  {
+    type: 'search',
+    name: 'Search',
+    icon: 'discoverApp',
+  },
+  {
+    type: 'visualization-visbuilder',
+    name: 'Visualization(visbuilder)',
+    icon: 'visualizeApp',
+  },
+];
+
+const render = renderFn(() => {
+  const services = getServices();
+  const navigateToUrl = services.application.navigateToUrl;
+  const recentAccessed = useObservable(services.chrome.recentlyAccessed.get$(), []);
```

Comment:
I suppose the correct order here should be:
1. Filter by type
2. Slice to no more than 8 items
3. render the items